node/subsystem-types/src/messages.rs

```diff
@@ -195,10 +195,16 @@ pub enum CollatorProtocolMessage {
 	/// This should be sent before any `DistributeCollation` message.
 	CollateOn(ParaId),
 	/// Provide a collation to distribute to validators with an optional result sender.
+	/// The second argument is the hash of parachain head before candidate's execution.
```

Comment:
'before candidate's execution' is kind of an unclear wording. 

This is the candidate's parent head-data hash, right?